lib/rucio/core/distance.py

```diff
@@ -141,3 +142,28 @@ def update_distances(src_rse_id=None, dest_rse_id=None, parameters=None, session
         query.update(params)
     except IntegrityError as error:
         raise exception.RucioException(error.args)
+
+
+@read_session
+def export_distances(session=None):
```

Comment:
def list_distances

Reply:
list_distances to list all distances works as well, :) no ? and looks more coherent with the current conventions (list_dids, list_rses, etc)

Reply:
list_distances can be also extended with a filters={} parameters that can be used later on.